packages/theme-wizard-website/public/templates/collage/variation.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html lang="nl-NL">
+  <head>
+    <title>Collage Variation</title>
+    <link rel="stylesheet" href="./variation.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@nl-design-system-candidate/paragraph-css@2/dist/paragraph.css"
```

Comment:
Note to self (future reviewer): de CDN moet nog vervangen worden door een interne dependency